tests/framework/parser.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"fmt"
+
+	yaml "gopkg.in/yaml.v2"
```

Comment:
3rd party imports at the bottom?

Reply:
I think in general we put them in that order? [Random example](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/apply/apply.go)

Reply:
oh, maybe I'm doing it wrong :/